README.md

```diff
@@ -118,6 +119,86 @@ aligned \
 --conn wss://batcher.alignedlayer.com
 ```
 
+## Register as an Aligned operator in testnet
+
+### Requirements
+
+> [!NOTE]
+> You must be whitelisted to become an Aligned operator.
+
+This guide assumes you are already [registered as an operator with EigenLayer](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation).
+
+Ensure you have the following installed:
+
+- [Go](https://go.dev/doc/install)
+- [Rust](https://www.rust-lang.org/tools/install)
+
+#### Install the Operator Binary
+
+To install the operator binary, run:
+
+```bash
+make build_operator
+```
+
+#### Update the operator
+
+To update the operator, first stop the process running the operator (if there is any) and then run:
+
+```bash
+make build_operator
+```
+
+#### Configuration
+
+Update the following placeholders in `./config-files/config-operator.yaml`:
+
+- `"<ecdsa_key_store_location_path>"`
+- `"<bls_key_store_location_path>"`
+- `"<operator_address>"`
+- `"<earnings_receiver_address>"`
+
```

Comment:
```suggestion
`<ecdsa_key_store_location_path>` and `<bls_key_store_location_path>` are the paths to your keys generated with the EigenLayer CLI, `<operator_address>` and `<earnings_receiver_address>` can be found in the `operator.yaml` file created in the EigenLayer registration process.
```

Reply:
Solved in [1602afd](https://github.com/yetanotherco/aligned_layer/pull/373/commits/1602afda804b9ef968d16156cd1fe6886154dcfa).